LayoutTestTests/IteratorTests.m

```diff
@@ -198,14 +198,15 @@ - (void)validateGenerators:(NSArray *)values context:(NSMutableSet *)context {
 
 // View Provider Protocol
 
-+ (NSDictionary *)dataSpecForTest {
++ (nullable NSDictionary *)dataSpecForTestWithError:(__unused NSError * _Nullable __autoreleasing *)error {
     return testData;
 }
 
 + (UIView *)viewForData:(__unused NSDictionary *)data
```

Comment:
If we end up going with `nullable` need to update that here and elsewhere

Reply:
aww nice catch, updated it here as well